stress_test/controller.py

```diff
@@ -0,0 +1,402 @@
+# Copyright (c) 2016 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+
+""" Controller Class- All controller-related functionality is here"""
+
+import json
+import common
+import logging
+import os
+import subprocess
+import time
+import util.netutil
+import util.process
+import conf_collections_util
+import requests
+import emulators.nb_generator.flow_utils
+
+
+
+class Controller:
+
+
+    def __init__(self, ctrl_base_dir, test_config):
+
+        """Create a Controller object. Options from JSON input file
+        :param test_config: JSON input configuration
+        :type test_config: JSON configuration dictionary
```

Comment:
When describing the types, I think we mention just the native Python type (e.g. `dict` in this case). Have a look at other docstrings to verify this.